### ISSUE TITLE
Revisit Install instructions

### ISSUE THREAD

**Issue by robingroppe** [alfg/guildbit#27]
Please revisit the install instructions.
It is not possible to follow the install instructions.
A lot seems to be missing there.

**Comment by alfg** [comment]
Hi @robingroppe,

I'm sure the installation documentation can be improved. Which steps do you feel are missing?

**Comment by robingroppe** [comment]
The initial dependecies are missing.
Where to Clone the Project?
Some steps are in the wrong Order, I.e. you cant Run easy_install before
its started.
Link the murmur config from Where to Where?
How to Start that thing?

**Comment by alfg** [comment]
Hi @robingroppe, are you referring to the development setup, or full install?

You can clone the project anywhere you would like as long as it has the proper permissions. Also, the murmur config is part of the [murmur-rest](https://github.com/alfg/murmur-rest) project, which has its own setup instructions.

You can start the guildbit development server with: `python runserver.py runserver`. Or for a production build you would need to use a wsgi server such as [gunicorn](http://gunicorn.org/). I have provided the [wsgi.py](https://github.com/alfg/guildbit/blob/master/wsgi.py) file for this.

I hope that helps. However, I'll leave this issue open as a reminder to improve the setup docs. :)

**Comment by robingroppe** [comment]
Okay I understand.
What I am missing is a complete guide from a minimal server install to finish.
For someone who is not really into this project, its hard to understand how this all works together.
From the docs i can read right now, I have to guess and try what will work.

**Comment by alfg** [comment]
I see. Thanks for the input and feedback! I'll look at updating the installation docs and adding a dockerfile in the near future.

**Comment by alfg** [comment]
Hi @robingroppe, 

Just a quick update. I've merged in https://github.com/alfg/guildbit/pull/28 with some updated installation instructions on the readme, as well as providing a Dockerfile and docker-compose file for creating a full working development environment, including the murmur-rest project.

Hopefully this helps understand the build process a bit more, especially if you're familiar with Docker. I'll also be adding some more detailed configuration guides on https://github.com/alfg/guildbit/wiki/Configuration-Guide (TODO for now). It'll be an ongoing effort, so this is just a start.

When I first started Guildbit, it was closed-source for about 2 years before going open-source. So most of the build instructions I had were a bit hacky that I didn't include here. I'll eventually clean them up and add them over into the wiki.

Thanks!
